ocs_ci/ocs/resources/pod.py

```diff
@@ -722,6 +722,28 @@ def get_csi_pod_log_details(self, logs_dir, log_file_name):
                 current_log_file_size = file_details[4]
         return gz_logs_num, current_log_file_size
 
+    def get_container_data(self, container_name):
+        """
+        Get the container data
```

Comment:
Get the container data for a requested container

Reply:
Done